scripts/is_lat_unit_matters.py

```diff
@@ -0,0 +1,90 @@
+﻿"""
+is_lat_unit_matters.py
+
+By Wedward Wei
+
+After I explore the data, I found some interesting detail. Like we have differnt types of "LongitudeUnits"
+I am wondering if the latitude data could be presented in differnt ways, so we meed more steps before plot.
+So I wrote this code to see if the worry is necessary.
+
+"""
+
+import os
+import sys
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+"""
+If data is not ready, we need download the raw data and unzip the files in the same route
+If data is ready, return the DataFrame
+"""
+def if_data_ready(filename):
+    if os.path.exists(filename):
+        print (filename+" already exist")
+        return pd.read_csv(filename)
+    else:
+        print (filename+" does not exist, please check the file")
+        sys.exit()
+file_path = os.getcwd()+"\\data\\CLIWOC15.csv"
+raw_data = if_data_ready(file_path)
+
+# Show that we have differnt types of "LongitudeUnits"
+print(pd.value_counts(raw_data["LongitudeUnits"])) 
+
+raw_data_360_1 = raw_data[raw_data["LongitudeUnits"]=="360 degrees"]
```

Comment:
It might be useful to use the outputs from pd.value_counts to create the separate raw data files. This way the function would be able to look at any type of longitude unit, not just the ones that you define.